src/resources/Sources/SourcesInterfaces.ts

```diff
@@ -249,6 +249,7 @@ export interface FormAuthenticationConfig {
     addFoundInputs?: boolean;
     authenticationFailed?: FormAuthenticationFailedConfiguration;
     enableJavaScript?: boolean;
+    forceLogin?: boolean;

```

Comment:
No test for this addition?

Reply:
I don't think it's necessary to test this addition. Normally, we test all the calls but not all the interfaces, for instance: https://github.com/coveo/platform-client/blob/master/src/resources/Sources/tests/Sources.spec.ts Another reason is this property is not officially documented yet, I see that it's used in the Backbone model that we want to replace, but I can't find it in the Swagger https://platformdev.cloud.coveo.com/docs/?urls.primaryName=Source#/Sources/rest_organizations_paramId_sources_paramId_get